helpers/cron-job/unlock-users.js

```diff
@@ -67,7 +67,7 @@ const unlockUsers = () =>
             afterChanges
           );
           await addHistoryRecord(historyRecord);
-          return sendEmail(userData.email, UNLOCK_USER);
```

Comment:
We are in for loop where each user that needs to be unlocked is iterating. Using return will break the loop, so I removed it.

Reply:
use Promise.all and map

Reply:
done